lib/tasks/maintenance.rake

```diff
@@ -0,0 +1,19 @@
+namespace :maintenance do
+  task prison_uuids: :environment do
+    mapping_path =
+      Rails.root.join('db', 'seeds', 'prisons', 'uuid_mappings.yml')
+
+    mappings = YAML.load(File.read(mapping_path)) || {}
+
+    Dir[Rails.root.join('db', 'seeds', 'prisons', '*.yml')].each do |path|
```

Comment:
It skips `uuid_mappings.yml`.  On reflection, this seems like a bit of a smell.  I'll move it up a level and rename it.